apps/src/studio/studio.js

```diff
@@ -3900,13 +3962,27 @@ Studio.endGame = function(opts) {
 
   if (winValue == "win") {
     Studio.trackedBehavior.hasWonGame = true;
-    Studio.setScoreText({text: studioMsg.winMessage()});
+    Studio.setVictoryText({text: studioMsg.winMessage()});
   } else if (winValue== "lose") {
     Studio.trackedBehavior.hasLostGame = true;
-    Studio.setScoreText({text: studioMsg.loseMessage()});
+    Studio.setVictoryText({text: studioMsg.loseMessage()});
   } else {
     throw new RangeError("Incorrect parameter: " + opts.value);
   }
+
+  Studio.gameState = Studio.GameStates.OVER;
+};
+
+Studio.onMouseDown = function (e) {
+  if (Studio.gameState === Studio.GameStates.WAITING) {
+    Studio.runButtonClick();
+  } else if (Studio.gameState === Studio.GameStates.OVER) {
+    // do a reset
+    var resetButton = document.getElementById('resetButton');
```

Comment:
Updated to use `studioApp.resetButtonClick()` which isn't a lot better, but doesn't directly depend on the DOM.